docs/modules/ROOT/pages/cicd_concept.adoc

```diff
@@ -10,21 +10,30 @@ A typical CI/CD framework would deploy images to the following three environment
 
 === Development
 
-The development environment is used for internal testing of infrastructure, such as endpoints,
-pipelines, back-end authentication, load tests, and database configuration. In order to be free
-to change or take down the development environment at will, we recommend against giving 
-external organizations any access to it.
+The development environment is used for internal testing of changes to infrastructure such as
+endpoints, pipelines, back-end authentication, load tests, and database configuration.
+
+Merge requests should deploy so-called 
+https://about.gitlab.com/blog/2016/11/22/introducing-review-apps[Review-Apps]
```

Comment:
Cool! Then we can omit the `-` and write "Review Apps". :+1:

Reply:
I'm inclined to have this written in lower-case, i.e. "review apps". What do you think?

Reply:
OK, that makes sense.